src/style.css

```diff
@@ -0,0 +1,78 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.form__big {
+  margin-top: 20px;
+  position: relative;
+}
+
+.form__small {
+  margin-top: 20px;
+  position: relative;
+}
```

Comment:
```suggestion
.form {
  margin-top: 20px;
  position: relative;
}
```

Reply:
No need to duplicate styles, just create one class and apply it to every element that should have this styles. Don't repeat yourself